File: src/main/com/rainsoft/spark/scala/BcpChat.scala
package com.rainsoft.spark.scala

import java.io.File

import org.apache.spark.{SparkConf, SparkContext}

import scala.io.Source

/**
  * Created by Administrator on 2017-04-07.
  */
object BcpChat {
    //    val filePath = "D:\\Program Files\\Java\\JetBrains\\workspace\\bigdata\\src\\main\\zdata\\bcp\\testdata"
    val filePath = "D:\\Program Files\\Java\\JetBrains\\workspace\\bigdata\\src\\main\\zdata\\bcp\\"

    def main(args: Array[String]): Unit = {
        val dir = new File(filePath)
        val fileList = dir.listFiles().filter(_.isFile).filter(_.toString.endsWith("bcp"))

        fileList.foreach(println)
        //        val data = convertFilContext(filePath)
        //        data.foreach(println)

    }


}

File: src/main/com/rainsoft/spark/java/BcpChat.java
package com.rainsoft.spark.java;

import org.apache.spark.SparkConf;
import org.apache.spark.SparkContext;

/**
 * Created by Administrator on 2017-04-07.
 */
public class BcpChat {
    public static void main(String[] args) {
        SparkConf conf = new SparkConf()
                .setAppName("bcp transform")
                .setMaster("local");

        SparkContext sc = new SparkContext(conf);


    }
}
